frontend/coprs_frontend/coprs/views/backend_ns/backend_general.py

```diff
@@ -143,7 +143,11 @@ def get_build_record(task, for_backend=False):
         "sandbox": task.build.sandbox,
         "background": bool(task.build.is_background),
         "chroot": task.mock_chroot.name,
-        "tags": task.mock_chroot.tags + task.tags,
+        "tags": task.mock_chroot.tags + task.tags + (
+            # TODO just for testing purposes, add a boolean
+            # `models.MockChroot.subscription` column to the database?
+            ["subscription"] if task.mock_chroot.name == "rhel-9-x86_64" else []
```

Comment:
I think the copr-rpmbuild part is done but I would like to have an ACK here before starting to add database columns.

My plan adding `models.MockChroot.subscription` column to the database. By default `False` and we would manually set it to `True` for all RHEL-based chroots. Then we would send a `subscription` tag like this ^^.

WDYT?